src/Server.php

```diff
@@ -68,12 +68,12 @@ public function startBrokerSession()
     {
         if (isset($this->brokerId)) return;
 
-        if (!isset($_GET['sso_session'])) {
+		$sid =  $this->getBrokerSessionID();
```

Comment:
Please adhere to [PSR2](http://www.php-fig.org/psr/psr-2/) and use 4 spaces, not tabs.